packages/ove-service-imagetiles/src/OVE.Service.ImageTiles/Startup.cs

```diff
@@ -155,6 +155,7 @@ public class Startup {
             app.UseSwagger()
                 .UseSwaggerUI(c => {
                     c.SwaggerEndpoint("/swagger/" + _version + "/swagger.json", "TileService " + _version);
+					c.RoutePrefix = "api-docs";

```

Comment:
Tabs instead of spaces, again.

```suggestion
                    c.RoutePrefix = "api-docs";
```